andela/wger_vulcan
# [Feature #142465271] Enable social logins

#### What does this PR do?
Add the social login feature

#### Description of Task to be completed? 
Enable the user to alternatively be able to login to Wger directly using their Twitter, Facebook or Google accounts.

#### How should this be manually tested? 
Run ```python manage.py runserver``` in the terminal.
On the wger dashboard, on the top left side click on  ```Login/Register``` and select ```Login``` from the drop down menu that appears.
Click on the icon of the social site of ypur preference in order for you to login.

#### Any background context you want to provide? 
None

#### What are the relevant pivotal tracker stories? 
#142465271

#### Screenshots (if appropriate) #### 
<img width="1280" alt="screen shot 2017-05-07 at 23 43 09" src="https://cloud.githubusercontent.com/assets/25659643/25785308/e5b94dda-3385-11e7-822f-11878921e3f3.png">


Questions:
None
